SKYKit.podspec

```diff
@@ -12,8 +12,6 @@ Pod::Spec.new do |s|
 
   s.platform     = :ios, '7.0'
 
-  s.default_subspec = 'Core'
```

Comment:
Oh, my bad, I didn't see this line actually.

Base on the discussion on https://github.com/SkygearIO/skygear-SDK-iOS/pull/97, I think I would include 'Core' and 'ForgotPassword' in default spec.

'SKYKit/Facebook' depends on 'FBSDKCoreKit', which I think is a bad idea to make skygear looks like it depends on facebook, given that most pod users would just `pod 'SKYKit'`.